web/src/components/TaskProgressCheckbox.tsx

```diff
@@ -83,6 +83,7 @@ export const TaskProgressCheckbox = (props: Props): ReactElement => {
     if (isTaxTask(props.taskId)) {
       if (nextStatus === "COMPLETED") {
         redirectOnSuccess = true;
+        analytics.event.tax_registration_snackbar.submit.show_tax_registration_success_snackbar();
```

Comment:
Are we firing this same event twice? Do we need both invocations of this function

Reply:
The event in `SidebarCardRegisteredForTaxesNudge.tsx` fires when one clicks the button on the nudge. The event in `TaskProgressCheckbox.tsx` fires when one sets the task to complete.